backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala

```diff
@@ -774,4 +775,27 @@ class TestOperator extends VeloxWholeStageTransformerSuite with AdaptiveSparkPla
       }
     }
   }
+
+  test("timestamp cast fallback") {
+    withTempPath {
+      path =>
+        (0 to 3).toDF("x").write.parquet(path.getCanonicalPath)
+        spark.read.parquet(path.getCanonicalPath).createOrReplaceTempView("view")
+        runQueryAndCompare(
+          "SELECT x FROM view WHERE cast(x as timestamp) " +
+            "IN ('1970-01-01 08:00:00.001','1970-01-01 08:00:00.2')") {
+          df =>
+            {
+              val plans = getExecutedPlan(df)
+              val result =
+                plans.exists(plan => plan.find(_.isInstanceOf[BatchScanExec]).isDefined) &&
```

Comment:
`BatchScanExec` Does this mean it is falling back to Spark?

Reply:
Maybe we don't need to verify a fallback query plan as offloaded execution should be the preferred.

Reply:
Ok, let me remove it.